assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -100,12 +100,18 @@ const setGuestAccountCreationToken = (guestAccountCreationToken: string): Action
 const setThankYouPageStage = (thankYouPageStage: ThankYouPageStage): Action =>
   ({ type: 'SET_THANK_YOU_PAGE_STAGE', thankYouPageStage });
 
-const setPaymentIsReady = (paymentReady: boolean, paymentHandlers: ?{ [PaymentMethod]: PaymentHandler }): Action =>
-  ({ type: 'UPDATE_PAYMENT_READY', paymentReady, paymentHandlers: paymentHandlers || null });
+const setThirdPartyPaymentLibrary =
```

Comment:
This is over several lines in order to make the linter happy. ✨ Solution thanks to @jranks123